Backend/testing/test_tts_integration.py

```
@@ -0,0 +1,103 @@
+        try:
+            response = requests.post(base_url, data={
+                "character_id": test["character_id"],
+                "user_text": test["user_text"]
+            })
+            
+            total_time = time.time() - start_time
+            
+            if response.status_code == 200:
+                result = response.json()
+                
+                print(f"   ✅ Status: Success")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)